tests/extensions/pdf/test_exporter.py

```diff
@@ -0,0 +1,93 @@
+import os
+import shutil
+
+import pytest
+
+from mfr.extensions.pdf import settings
+from mfr.extensions.pdf import exceptions
+from mfr.extensions.pdf import PdfExporter
+
+BASE = os.path.dirname(os.path.abspath(__file__))
+
+
+@pytest.fixture
+def directory(tmpdir):
+    return str(tmpdir)
+
+
+@pytest.fixture(scope="function", autouse=True)
+def setup_filesystem(directory):
+    shutil.rmtree(directory)
+    os.makedirs(directory, exist_ok=True)
+
+
+class TestPdfExporter:
+    '''
+    Opening and veryifying pdfs with report lab is a paid feature
+    so these tests are mostly just to see if the file gets created
+    in the correct place
```

Comment:
Is there anything else in the stdlib that can assert that the output is actually a pdf?  If not, no worries, it would just be good to assert that a pdf is produced, rather than e.g. and html error message.

Reply:
Followup: it looks like PDFs have a signature: https://en.wikipedia.org/wiki/List_of_file_signatures

Reply:
This is great, implementing

Reply:
I found some libraries that use system libraries to check things (libmagic used by python-magic) etc. They would require adding things to the docker file or something like that. I just went with opening the file, and pulling the signature out directly and looking at it. Slightly crude but works fine.

Reply:
:+1: